### ISSUE TITLE
sc-mb-import: Button appears but no functionality

### ISSUE THREAD

**Issue by Byter09** [garylaski/userscripts#9]
Hello! I'm trying to import songs from an artist, for example on https://soundcloud.com/kutemane/sets/dont-leave-me, and I can see the Import button, but it does absolutely nothing when I click on it. I am logged in on SoundCloud.

Versions:
```
Browser: Mozilla Firefox 113.0.2 20230523141515 20230523141515
Userscript Addon: Tampermonkey 4.18.1
OS: Arch Linux x86_64 6.3.4-zen1-1-zen
```

I don't quite know what other information could be useful to investigate this, but I'll try to provide everything you need. I can also run tests if you need me to.

Hope I'm not just doing something stupid :D

EDIT: Also, not sure if this is related, but the import button is available on the two track sites for the songs in that album, but if I go to a song that is in no album, the button doesn't even appear. So, this works (as in, the button appears, it still doesn't open a new tab) https://soundcloud.com/kutemane/remorse (it's in the "Don't Leave Me" album), but this doesn't: https://soundcloud.com/kutemane/face-to-face

**Comment by garylaski** [comment]
Thanks for reporting this, you are not doing anything stupid. It looks there are a few things wrong:

> https://soundcloud.com/kutemane/sets/dont-leave-me

It looks like Soundcloud changed their API for all albums/sets. I'll have to update the code.  #10 

> https://soundcloud.com/kutemane/remorse

This is the case where there is a track that is part of a release.  These should not be submitted to Musicbrainz as individual tracks at all, so I just returned. #11 

> https://soundcloud.com/kutemane/face-to-face

This is an alternate layout, so the import button cannot find where it is supposed to be placed. #12 

I've opened issues for all these cases.
